packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -381,12 +387,22 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 	purchasedThemes,
 } ) => {
 	const hasCategories = !! categorization?.categories.length;
+	const blankCanvasDesign = {
+		recipe: {
+			stylesheet: 'pub/blank-canvas-blocks',
+		},
+		slug: 'blank-canvas-blocks',
+		title: 'Blank Canvas',
+	} as Design;
+
 	const filteredDesigns = useMemo( () => {
 		const result = categorization?.selection
 			? filterDesignsByCategory( designs, categorization.selection )
 			: designs.slice(); // cloning because otherwise .sort() would mutate the original prop
 
 		result.sort( sortDesigns );
+		result.splice( Math.min( result.length, 3 ), 0, blankCanvasDesign );
```

Comment:
We might need to check feature flag `design-picker-pattern-assembler` here or in `packages/design-picker/src/components/unified-design-picker.tsx`